test/integration/good/parser-generator/pretty.expected

```diff
@@ -107,6 +107,13 @@ transformed data {
   print("expect: [3,7,7,8,5]  v: ", v);
 }
 
+  $ ../../../../../install/default/bin/stanc --auto-format fun-args-trailing-comma.stan
+functions {
+  void foo(real x, real y) {
```

Comment:
I wonder how difficult it would be to preserve these commas through pretty-printing. If they idea is that this better supports a git workflow, we probably don't want it to be counter to using the auto-formatter.

Also, can you add some comments between/after the commas here to see what happens when pretty-printing?